### PR TITLE
adding container instance facts

### PR DESCRIPTION
**Pull request by zikalino**
##### SUMMARY
Adding missing container instance facts

##### ISSUE TYPE
 - New Module Pull Request

##### COMPONENT NAME
azure_rm_containerinstance_facts

##### ANSIBLE VERSION
2.6

### CODE REVIEW

**Comment by Fred-sun** [comment]
@zikalino does this PR need to review or wait other change plan? could you help to update the comment? Thanks!

**Comment by Fred-sun** [comment]
@zikalino does this PR ready for review, please help confirm? Thanks!

**Comment by zikalino** [comment]
it's still [WIP] so needs a few more things

**Comment by Fred-sun** [comment]
@zikalino Thanks for your update, after you finished, I will push to review!

**Comment by Fred-sun** [comment]
@zikalino Could you help confirm this PR ready for review or not? Thanks!

**Comment by Fred-sun** [comment]
@zikalino Could you help confirm the PR ready for review or not? Thanks!

File: lib/ansible/modules/cloud/azure/azure_rm_containerinstance_facts.py [hunk 1]
```diff
@@ -0,0 +1,280 @@
...
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: azure_rm_containerinstance_facts
+version_added: "2.7"
```
Comment (yungezz) [inline lib/ansible/modules/cloud/azure/azure_rm_containerinstance_facts.py]: 2.8

File: lib/ansible/modules/cloud/azure/azure_rm_containerinstance_facts.py [hunk 2]
```diff
@@ -0,0 +1,280 @@
...
+short_description: Get Container Instance facts.
```
Comment (yungezz) [inline lib/ansible/modules/cloud/azure/azure_rm_containerinstance_facts.py]: Azure

File: lib/ansible/modules/cloud/azure/azure_rm_containerinstance_facts.py [hunk 3]
```diff
@@ -0,0 +1,280 @@
...
+                - The containers within the container group.
+            returned: always
+            type: complex
+            sample: containers
+            contains:
+                name:
+                    description:
+                        - The name of the container instance.
+                    returned: always
+                    type: str
```
Comment (yungezz) [inline lib/ansible/modules/cloud/azure/azure_rm_containerinstance_facts.py:115]: remove real subs id

**Comment by zikalino** [comment]
@yungezz all comments fixed now

**Review by yungezz — APPROVED** [review]

**Pull request merged**